samples/apps/forum/src/controllers/poll.ts

```diff
@@ -71,7 +70,8 @@ interface GetPollsResponse {
 
 // Export REST API request/response types for unit tests
 export {
```

Comment:
Do we use an autoformatter for TS? If not, should we pick one and hook it up to `ci-checks.sh`?